docs/Development/Theme.md

```diff
@@ -2027,3 +2027,259 @@ CSS file:
 .action-button:active:hover, .action-button:active:focus, .action-button:active.focus, .action-button.active:hover, .action-button.active:focus, .action-button.active.focus, .open>.action-button.dropdown-toggle:hover, .open>.action-button.dropdown-toggle:focus, .open>.action-button.dropdown-toggle.focus {    color: #fff;    background-color: #3aa821;    border-color: #aa0303;}
 ```
 </details>
+
+---
+
+### [Bitcoin People](https://bitcoinpeople.it)
+
+![Bitcoin People checkout](../img/Theme-BitcoinPeople.png)
```

Comment:
@davexbtc you need to add the image to your pull request, as Pavlenex already mentioned.

You can see your image is broken in the document preview: https://github.com/btcpayserver/btcpayserver-doc/blob/90a3fad76a436a4c5b7ff382a74aea5d710feacf/docs/Development/Theme.md#bitcoin-people

Let me know if you don't know how to do that and I can explain further. Your Pull Request will not be merged until you complete this.

Reply:
I think I've done.

Reply:
I made the pull request 1014.

Reply:
Added it here.